tokens/vivo.json

```diff
@@ -125,26 +125,53 @@
       "type": "color",
       "description": "vivoPurpleLight50"
     },
-    "buttonSecondaryBackground": {
+    "buttonSecondaryBorder": {
       "value": "{palette.vivoPurple}",
       "type": "color",
       "description": "vivoPurple"
     },
-    "buttonSecondaryBackgroundSelected": {
+  
+    "buttonSecondaryBorderSelected": {
+      "value": "{palette.vivoPurpleDark}",
+      "type": "color",
+      "description": "vivoPurpleDark"
+    },
+
+    "buttonSecondaryBackgroundHover": {
       "value": "{palette.vivoPurpleDark}",
```

Comment:
is it correct? maybe too dark for button background hover

Reply:
Absolutely right, thanks!